app/templates/package.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "",
+	"version": "0.0.0",
+	"description": "",
+	"homepage": "",
+	"bugs": "",
+	"author": {
+		"name": "",
+		"email": "",
+		"url": ""
+	},
+	"main": "index.js",
+	"engines": {
+		"node": ">=0.8.0"
+	}
+}
```

Comment:
use this package.json instead https://github.com/sindresorhus/node-module-boilerplate/blob/master/boilerplate/package.json